cashctrl_ledger/tests/test_accounts.py

```diff
@@ -14,165 +14,139 @@
-        return initial_ledger
-
-    # TODO: Move this test to the PyLedger package when storing Price history implemented
-    @pytest.mark.parametrize(
-        "account, date, expected",
-        [
-            (9991, "2024-02-21", {"USD": 0.00, "reporting_currency": 0.00}),
-            (9991, "2024-02-22", {"USD": 100.00, "reporting_currency": 88.88}),
-            (9991, "2024-03-23", {"USD": 200.00, "reporting_currency": 174.43}),
-            (9991, "2024-04-24", {"USD": 100.00, "reporting_currency": 96.66}),
-            (9991, "2024-06-26", {"USD": 0.00, "reporting_currency": 1.11}),
-            (9991, None, {"USD": 0.00, "reporting_currency": 1.11}),
-            (9992, "2024-01-20", {"CHF": 0.00, "reporting_currency": 0.00}),
-            (9992, "2024-01-21", {"CHF": 100.00, "reporting_currency": 100.00}),
-            (9992, "2024-05-25", {"CHF": 110.00, "reporting_currency": 110.00}),
-            (9992, None, {"CHF": 110.00, "reporting_currency": 110.00}),
-            (9994, "2024-02-21", {"USD": 0.00, "reporting_currency": 0.00}),
-            (9994, "2024-02-22", {"USD": -100.00, "reporting_currency": -88.88}),
-            (9994, "2024-04-24", {"USD": 0.00, "reporting_currency": -11.11}),
-            (9994, None, {"USD": 0.00, "reporting_currency": -11.11}),
-            (9995, "2024-01-20", {"CHF": 0.00, "reporting_currency": 0.00}),
-            (9995, "2024-01-21", {"CHF": -100.00, "reporting_currency": -100.00}),
-            (9995, "2024-03-23", {"CHF": -185.55, "reporting_currency": -185.55}),
-            (9995, "2024-05-25", {"CHF": -195.55, "reporting_currency": -195.55}),
-            (9995, "2024-06-26", {"CHF": -100.00, "reporting_currency": -100.00}),
-            (9995, None, {"CHF": -100.00, "reporting_currency": -100.00}),
-        ],
-    )
-    def test_account_single_balance(self, ledger_with_balance, account, date, expected):
-        balance = ledger_with_balance._single_account_balance(account=account, date=date)
-        assert balance == expected
-
-    def test_add_already_existed_raise_error(self, ledger):
-        super().test_add_already_existed_raise_error(
-            ledger, error_class=RequestException, error_message="This number is already used"
+    def engine(self, initial_engine):
+        initial_engine.clear()
+        return initial_engine
+
+    def test_account_accessor_mutators(self, engine):
+        super().test_account_accessor_mutators(engine, ignore_row_order=True)
+
+    def test_add_existing_account_raise_error(self, engine):
+        super().test_add_existing_account_raise_error(
+            engine, error_class=RequestException, error_message="This number is already used"
         )
 
-    def test_modify_non_existed_raise_error(self, ledger):
-        super().test_modify_non_existed_raise_error(
-            ledger, error_class=ValueError, error_message="No id found for account"
+    def test_add_existing_account_raise_error(self, engine):
```

Comment:
Why can we not rely on the parent class method here?

Reply:
This was added by mistake, need to remove

Reply:
Understood. Thanks. 

Please add your explanation as a comment or doc string to motivate why we override the test.
I am suer I won't remember two months down the road.